teku/src/main/java/tech/pegasys/teku/cli/subcommand/TransitionCommand.java

```diff
@@ -181,11 +182,22 @@ private BeaconState readState(final Spec spec, final Bytes inData) {
   }
 
   private SignedBeaconBlock readBlock(final Spec spec, final String path) throws IOException {
-    final Bytes blockData = Bytes.wrap(Files.readAllBytes(Path.of(path)));
+    final byte[] blockDataBytesArray = Files.readAllBytes(Path.of(path));
     try {
+      return spec.deserializeSignedBeaconBlock(Bytes.wrap(blockDataBytesArray));
+    } catch (final RuntimeException e) {
+      return deserializeSignedBeaconBlockFromHex(spec, path, blockDataBytesArray);
+    }
+  }
+
+  private SignedBeaconBlock deserializeSignedBeaconBlockFromHex(
+      final Spec spec, final String path, final byte[] hexBlockData) {
+    try {
+      final Bytes blockData = Bytes.wrap(Hex.decode(hexBlockData));
       return spec.deserializeSignedBeaconBlock(blockData);
-    } catch (final IllegalArgumentException e) {
-      throw new SSZException("Failed to parse SSZ (" + path + "): " + e.getMessage(), e);
+    } catch (final RuntimeException e) {
+      throw new RuntimeException(
+          String.format("Failed to parse SSZ/HEX (%s): %s", path, e.getMessage()), e);
```

Comment:
I suspect the exception message we're getting here isn't the most useful one. ie if the SSZ was invalid (from the wrong fork or something) this would just give a message about an invalid hex character which is likely to be confusing.  We probably want to track the original exception from parsing the SSZ and provide both error messages here.

Reply:
I propagated down the SSZ parsing exception and added its message to the final exception